doc/CORTX_on_Open_Virtual_Appliance.rst

```diff
@@ -310,6 +308,8 @@ Restart CORTX
    
 Tested by:
 
+- Mar 25, 2021: Mukul Malhotra (mukul.malhotra@seagate.com) using OVA release 1.0.3 on Windows 10 running Oracle VirtualBox & VMware Workstation 6.1.16.
```

Comment:
@mukul-seagate11 @TechWriter-Mayur , the PR is being done by Mayur but the tested by says Mukul.  That's fine.  I'm just curious.  Mukul did the testing buy Mayur is adding the document?  Mayur, can you also do the testing yourself?